package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
Seems like this file shouldn't be in the PR, please check

Reply:
You're right, I miss remove it after trying install prompt-plus library. Will remove it. Thanks!

Reply:
Checked and is not a bad change, seems like the last update I miss to update package lock json with the new package json version